contributing/testing-best-practices.md

```diff
@@ -0,0 +1,61 @@
+
+# Testing Best Practices
+
+## Types of test
+
+- Unit tests: In Terraform terminology they refer to tests that [validate a resource schema](https://developer.hashicorp.com/terraform/plugin/framework/handling-data/schemas#unit-testing). That is done automatically [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/internal/provider/provider_test.go) for all resources and data sources using Terraform Framework Plugin. Additionally, we have general unit testing for testing a resource or unit without calling the external systems like the Atlas Go SDK.
+- [Acceptance (acc) tests](https://developer.hashicorp.com/terraform/plugin/testing/acceptance-tests): In Terraform terminology they refer to the use of real Terraform configurations to exercise the code in plan, apply, refresh, and destroy life cycles (real infrastructure resources are created as part of the test).
```

Comment:
feels a bit weird these two the link is the heading but unit is in the body

Reply:
changed, thx